torch_frame/nn/models/excelformer.py

```diff
@@ -71,8 +102,8 @@ class ExcelFormer(Module):
 
     ExcelFormer first converts the categorical features with a target
     statistics encoder into numerical features. Then it sorts the
-    numerical features with mutual information sort. So the model
-    itself limits to numerical features.
+    numerical features with mutual information sort. For categorical
```

Comment:
I think it's better to keep the line that the model limits to numerical features and if you'd need categoricals you need to convert them first.

Reply:
Yes, I agree and have kept it.